motorway/mixins.py

```diff
@@ -44,12 +48,23 @@ def send_message(self, message, process_id, time_consumed=None, sender=None, con
         except GroupingValueMissing:
             raise GroupingValueMissing("Message '%s' provided an invalid grouping_value: '%s'" % (message.content, message.grouping_value))
         for destination in socket_addresses:
-            message.send(
-                self.send_socks[destination],
-                process_id
-            )
+            # Keep trying to send the message until successful
+            message_retries = 0
+            while True:
+                try:
+                    message.send(
```

Comment:
Don't think we're doing 80 chars here, so no point to do a new line here, I guess?